docs/getting-started.md

```diff
@@ -102,7 +104,7 @@ helm install argo-artifacts stable/minio \
 
 Login to the Minio UI using a web browser (port 9000) after exposing obtaining the external IP using `kubectl`.
 ```sh
-kubectl get service argo-artifacts -o wide
+kubectl get service argo-artifacts -o wide -n argo
```

Comment:
This assumes the user has installed Minio into the Argo namespace, which isn't a safe assumption. Not that it's wrong, but we should add context here.

Reply:
I agree- I think the guide should probably be for installing into the `argo` namespace, but should state this at the start.

Reply:
Thank you for the comment @ntwrkguru . I'm trying to update the document to have everything installed into the `argo` namespace as stated at the very beginning. Just to clarify, where should this be clarified? Like
```
5. Install an Artifact Repository
...
Install Minio into the argo namespace:
```
?

Reply:
Maybe a blurb, like:

     Assuming Minio was installed into the "argo" namespace, the following `kubectl` command will expose the Minio UI. Login to the Minio UI using a web browser (port 9000).

Reply:
Thank you. Pushed one commit. I assume PRs will be squashed merged? Or should I squash myself?

Reply:
@alexec Were there other "getting started" guides that need to be addressed? Ideally, there would be 1 guide that then could link into the examples or focus areas for more details.

Reply:
I think links could be useful. Just a summary here in case it's helpful. The document currently has
* https://github.com/argoproj/argo/tree/stable/manifests
* https://github.com/argoproj/argo/blob/master/docs/workflow-rbac.md
* https://github.com/argoproj/argo/blob/master/examples/README.md
* https://github.com/argoproj/argo/blob/master/docs/configure-artifact-repository.md
* https://github.com/argoproj/argo/blob/master/docs/argo-server.md

Are there any other documents useful for getting-started?

Reply:
Not that really stand out. I'll follow the guide later and see if I can offer suggestions.